packages/node-sdk/src/apis/DefaultApi.ts

```
@@ -229,6 +249,61 @@ export class DefaultApi extends runtime.BaseAPI {
+    if (requestParameters["targetId"] == null) {
+      throw new runtime.RequiredError(
+        "targetId",
+        'Required parameter "targetId" was null or undefined when calling deleteTarget().',
+      );
+    }
+
+    const queryParameters: any = {};
+
+    const headerParameters: runtime.HTTPHeaders = {};
+
+    if (this.configuration && this.configuration.apiKey) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)